graphql_compiler/tests/test_input_data.py

```diff
@@ -1399,6 +1399,27 @@ def starts_with_op_filter():  # noqa: D103
         type_equivalence_hints=None)
 
 
+def starts_with_op_filter_no_value():  # noqa: D103
+    graphql_input = '''{
+        Animal {
+            name @filter(op_name: "starts_with")
```

Comment:
Is this a legal input? It doesn't seem like it to me.

Reply:
Yes - this test cases is meant to create an error during compilation

Reply:
Sorry, legal = compiles without errors. All inputs in this file are expected to compile correctly. `test_ir_generation_errors.py` is for inputs that are expected to raise errors.